fastly/vcl/polyfill-service.vcl

```diff
@@ -81,6 +81,39 @@ sub vcl_recv {
 		call breadcrumb_recv;
 	}
 
+
+	declare local var.compute-at-edge-active BOOL;
+	if (std.atoi(table.lookup(compute_at_edge_config, "active", "0")) == 1) {
+		set var.compute-at-edge-active = true;
+	} else {
+		set var.compute-at-edge-active = false;
+	}
+	if (var.compute-at-edge-active) {
+		# if the querystring parameter `use-compute-at-edge-backend` is set to yes`
+		# then use the c@e backend.
+		# else if the `use-compute-at-edge-backend` is not set to `no` then make
+		# 1 in 1000 requests which come directly from a client use the c@e backend
+		declare local var.use-compute-at-edge-backend STRING;
+		set var.use-compute-at-edge-backend = querystring.get(req.url, "use-compute-at-edge-backend");
+		if (var.use-compute-at-edge-backend == "yes" && (std.prefixof(req.url.path, "/v3/polyfill.js") || std.prefixof(req.url.path, "/v3/polyfill.min.js"))) {
+			set req.backend = F_compute_at_edge;
+			if (req.backend.healthy) {
+				# if using the c@e backend then do not use the cache at all
+				return(pass);
+			}
+		} else if (var.use-compute-at-edge-backend != "no") {
```

Comment:
```suggestion
        # requests can set the querystring parameter to `no` to avoid ever user compute-at-edge
        # if the querystring parameter is not set to no, then we run some logic to decide whether
        # to use compute-at-edge for this request or not.
		} else if (var.use-compute-at-edge-backend != "no") {
```